guidelines/internal-guidelines/new-repository-settings.md

```diff
@@ -0,0 +1,93 @@
+
+In Kyma, these rules are defined in the Prow [`config.yaml`](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml) file and handled by a Prow component called [Branch Protector](https://github.com/kyma-project/test-infra/blob/master/docs/prow/prow-architecture.md#branch-protector).
+
+If you add a new repository in:
+- `kyma-project`, you do not need to add a new entry to the Prow `config.yaml` file as the branch protection is already defined for [all repositories](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml#L380) within this organization. The only exception is if you want to specify additional rules that are not handled by Prow.
+- `kyma-incubator`, add a new repository entry to the Prow `config.yaml` file, under **branch-protection.kyma-incubator.repos**. [See](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml) an example of such an entry for the `marketplaces` repository.
+
+## Add webhooks
+
+Add ZenHub and Prow webhooks to integrate them with your repository. This way, GitHub can send all events from the repository to Prow and notify ZenHub on the selected events related to issues and pull requests.
+
+These settings are available under **Webhooks** in the left menu, under repository **Settings**:
+
+![Webhooks](../../assets/webhooks.png)
+
+### Prow webhook
+
+Only Prow admins can activate the Prow webhook. Contact them by raising an issue in the `test-infra` repository with details of your request. You can also contact them on the [CI Slack channel](https://kyma-community.slack.com/messages/CD1C8PK1B/).
+
+### ZenHub webhook
+
+To activate the ZenHub webhook:
+1. Go to the [Kyma workspace](https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/board?repos=139590616,139590577,139847219,139590666,139590641,147495537,151691065,146318286,157188288,156510564,167399060,169101295,171673731,150745068,167146343,165843160,170300585,186589820,188835954) on ZenHub.
+2. Click **Repos** and select the **Add Repos** option.
+
+![Zenhub webhook](../../assets/zenhub-webhook.png)
+
+3. Choose the organization in the left bottom corner of the pop-up box, select the repository you want to add from the available list, and click the **Add this repo** button.
+
+![Zenhub webhook 2](../../assets/zenhub-webhook-2.png)
+
+## Add a milv file
+
+If you define any governance-related [Prow job](https://github.com/kyma-project/test-infra/blob/master/prow/jobs/) for the new repository to validate documentation links, you must add a `milv.config.yaml` file at the root of the repository. [See](https://github.com/kyma-project/test-infra/blob/master/milv.config.yaml) an example of the milv file.
+
+## Create labels
+
+[Define labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories.
```

Comment:
We could link to the issues-workflow.md ---> https://github.com/kyma-project/community/blob/master/issues-workflow.md#default-labels

Reply:
I will link to the labels in the `kyma` repository instead -> https://github.com/kyma-project/kyma/labels. As discussed, the `issues-workflow.md` doc will need to be updated with exact recommendations for the label colors. So far, I guess that "real" repo example would be better.